test/verify/check-storage-stratis

```diff
@@ -161,6 +161,20 @@ class TestStorageStratis(StorageCase):
         self.confirm()
         b.wait_not_in_text("#detail-content", "fsys2-copy")
 
+        # Make an unmounted copy of the filesystem
```

Comment:
Let's only have one commit with changes to both the code and the tests.

Reply:
Alright, I'll squash when the rest of the changes are done :+1: